lms/static/js/verify_student/pay_and_verify.js

```diff
@@ -55,6 +55,8 @@ var edx = edx || {};
                     function(isVisible) { return isVisible; }
                 ),
                 upgrade: el.data('msg-key') === 'upgrade',
+                currencyCode: el.data('default-currency-code'),
```

Comment:
@rlucioni have made this change but not entirely sure why it was necessary. Out of interest sakes, what was the difference between calling toUpperCase there or in the template?